test/integration/cartridge_health_test.lua

```diff
@@ -38,3 +38,59 @@ g.test_cartridge_health_fail_handler = function()
         box.info = _G.old_info
     ]])
 end
+
+g.test_cartridge_health_handler_state_1 = function()
+    helpers.skip_cartridge_version_less('2.0.2')
+    helpers.upload_default_metrics_config(g.cluster)
+    local main_server = g.cluster:server('main')
+    main_server.net_box:eval([[
+        _G.old_info = box.info
+        box.info = {
+            status = 'alive',
```

Comment:
Status and state are not a box.info parameters. You can't use it like this, you should patch `membership.myself()` call in tests.

Reply:
You are right. Thank you!